modules/documents/client/views/partials/modal-document-confirm-publish.html

```diff
@@ -3,10 +3,12 @@
 		<button class="btn btn-default close" type="button" ng-click="confirmDlg.cancel()">
 			<span aria-hidden="true">&times;</span>
 		</button>
-		<h3 class="modal-title">Confirm Publish File(s)</h3>
+		<h3 ng-if="confirmDlg.publishableFiles.length === 0">No files can be published</h3>
```

Comment:
Please clarify "filters".  A angular filter can do text transform but it seems that we'd need a filter per text element.   This will move the display text out of the html and hide it in JS file that provides the angular filter.   Is this the ask here?